report_thesis/src/sections/introduction.tex

```diff
@@ -5,30 +5,42 @@ \section{Introduction}\label{sec:introduction}
 We define accuracy as the ability of a model to predict the composition of major oxides in Martian geological samples, while robustness refers to the stability of these predictions across different samples and oxides.
 
-We investigate various techniques to handle the high dimensionality, non-linearity, and small dataset size inherent in this problem, and evaluate the performance of these models using appropriate metrics. 
-Through extensive experiments on \gls{libs} data, we demonstrate the superior performance of our approach compared to existing methods in terms of both prediction accuracy and computational efficiency.
+To achieve these objectives, this research systematically explores a range of promising machine learning models and preprocessing techniques, identified through an extensive literature review and guided by a curiosity to explore unconventional approaches.
+Specifically, we implemented an experimental framework using an automated hyperparameter optimization tool to determine the most effective combinations of preprocessing methods and models for each major oxide analyzed.
+We began by evaluating various preprocessing techniques to understand their impact on model performance, selecting those that demonstrated the highest impact on improving the performance of each model.
+Following the preprocessing, the most promising models underwent an optimization process, allowing us to precisely tune both model configurations and their respective hyperparameters on a per-oxide basis, ensuring optimal performance tailored to the specific data characteristics of each oxide.
+Once the best configurations were identified, a stacking ensemble method was employed to create a meta learner for each oxide, significantly enhancing prediction accuracy and robustness beyond the capabilities of individual models.
+Through extensive experiments on \gls{libs} data, we systematically assessed and demonstrated the superior performance of our approach compared to existing methods, focusing on significant improvements in prediction accuracy and robustness.
 
 Our key contributions are as follows:
 \begin{itemize}
-    \item We develop a novel machine learning pipeline that effectively handles the challenges of \gls{libs} data to accurately predict major oxide compositions in Martian soil samples.
-    \item We conduct a comprehensive evaluation of various dimensionality reduction techniques and machine learning models to identify the optimal combination for this task. 
-	\item We demonstrate the superior performance of our approach compared to existing methods through extensive experiments on \gls{libs} data.
+    \item We develop a novel machine learning pipeline that effectively handles the challenges of LIBS data to accurately predict major oxide compositions in Martian soil samples.
```

Comment:
```suggestion
    \item We develop a novel machine learning pipeline that effectively handles the challenges of \gls{libs} data to accurately predict major oxide compositions in Martian soil samples.
```

Reply:
If we say `effectively handles`, we probably also need to prove it. How? We can show better accuracy and robustness, but the other one is more challenging. Especially given our (lack of) knowledge in non-CS fields here...

Reply:
```
\item We develop novel machine learning pipeline that demonstrates improved accuracy and robustness in predicting major oxide compositions in LIBS calibration target.
``